THANKS.txt

```diff
@@ -234,6 +234,8 @@ James Wright for simple documentation fixes
 Sam Wallan for scipy.linalg.lapack enhancements
 Richard Weiss for a bug fix in scipy.optimize._differentialevolution.py.
 Luigi F. Cruz for adding time/frequency domain option to scipy.signal.resample.
+Yili Hong, Mika Straka, Hamel Ajay Kothari and Maxi Marufo for adding Poisson
```

Comment:
Can these lines be removed from the PR? 
See [thread](https://mail.python.org/pipermail/scipy-dev/2020-August/024331.html) for discussion (changes to this file causes merge conflicts adding effort to contributors and maintainers). 
For now, I've opened a [wiki](https://github.com/scipy/scipy/wiki/*THANKS.txt-additions-modifications-for-1.6.0*)
If/once the PR is merged, we add thanks to the wiki. Similar process to what we followed in 1.5.0 release.